grpc-gcp/test/codegen.js

```diff
@@ -0,0 +1,2 @@
+const shell = require('shelljs');
+shell.exec(__dirname + '/codegen.sh');
```

Comment:
The imported module is `shx`, not `shelljs` - what's up with the import path here?